shape_completion_training/src/shape_completion_training/model/utils.py

```diff
@@ -69,3 +70,21 @@ def set_gpu_with_lowest_memory():
         return
     os.environ["CUDA_VISIBLE_DEVICES"] = best_gpu
 
+
+def reduce_mean_dict(dict):
```

Comment:
Another way to write this:

```
def reduce_dict(dict, reduce_fn=tf.reduce_mean):
  return {k: reduce_fn(v) for k, v in dict.items()}
```

Reply:
totally agreed. you may be shocked to know that I learned about dictionary comprehensions like three weeks ago, and this was copied from somewhere else.